domains/*.ultrix.is-cool.dev.json

```diff
@@ -0,0 +1,14 @@
+{
+    "domain": "is-cool.dev",
+    "subdomain": "*.ultrix",
+
+    "owner": {
+        "email": "pollardjackson@outlook.com"
+    },
+
+    "record": {
+        "CNAME": "not-setup-yet"
```

Comment:
You need to provide a valid CNAME. Domain reserving is not allowed.

Reply:
How would I do that for a wild card domain

Reply:
Also please check this out: https://github.com/is-a-dev/register/pull/8429